kibbeh/src/ui/NotificationElement/FollowNotification.tsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import { GenericNotification } from "./GenericNotification";
+import { SingleUser } from "../UserAvatar/SingleUser";
+import { Button } from "../Button";
+
+export interface FollowNotificationProps {
+  userAvatarSrc: string;
+  username: string;
+  userProfileLink?: string;
+  time: string;
+  isOnline?: boolean;
+  followed?: boolean;
+}
+
+export const FollowNotification: React.FC<FollowNotificationProps> = ({
+  userAvatarSrc,
+  isOnline = false,
+  username,
+  userProfileLink,
+  time,
+  followed = false,
+}) => {
+  const icon = <SingleUser src={userAvatarSrc} size="sm" isOnline={isOnline} />;
+
+  const notificationMsg = (
+    <>
+      <a
+        className="font-bold"
+        {...(userProfileLink ? { href: userProfileLink } : {})}
+      >
+        {username}&nbsp;
```

Comment:
Move all the whitespaces to the other strings, just in case we have to add some hover effects to the username in the future (such as underline).